### PR TITLE
[#1098] Apply service access model

### LINKED ISSUE #1098: Apply service access model
**Is your feature request related to a problem? Please describe.**

There is no description of the service access type, so the user doesn’t know if the below service is free/ open for all. How to access it, what should be done? 

**Describe the solution you'd like**

The service access model ( the correct naming to be agreed) should be more visible to the user. 
We need to plan how to show it to the user, things to project:

- [ ]  Information about the Service model should be visible in the Service list 

- [ ] Information about the Service model should be visible in the Service view 

- [ ] Be able to Filter by Service model ( to decide on the Titles that should be shown to the user).

### PR DESCRIPTION
**Pull request by goreck888**
Add backend and frontend markups for service type
Add service type filter written as order type
for future compatibility
Fixes #1098

### CODE REVIEW

**Comment by jarekzet** [comment]
Added styling for access type badges on service list and service details views.

File: app/helpers/service_helper.rb [hunk 1]
```diff
@@ -48,6 +48,10 @@ def providers_text(service)
     service.providers.map { |target| target.name }
   end
 
+  def order_type(service)
+    service&.service_type || service&.offers&.first&.offer_type || nil
```
Comment (bwilk) [inline app/helpers/service_helper.rb:52]: For the services having offers with different access types "VARIOUS ACCESS TYPES" should be displayed both on the service list and service view

Comment (goreck888) [reply app/helpers/service_helper.rb:52]: Ok, I'll wait for @jarekzet css mods. All my changes are stashed locally now.

**Comment by MinLidia** [comment]
@goreck888 In the searching fields the name Service type should be changed to Access Type

**Comment by MinLidia** [comment]
I have bug when filtering for External access type 
Screenshot 2020-07-23 at 12 11 00

File: app/models/filter/order_type.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Filter::OrderType < Filter
+  def initialize(params = {})
+    super(params: params.fetch(:params, {}),
+          field_name: "service_type",
```
Comment (martaswiatkowska) [inline app/models/filter/order_type.rb:6]: service_type? not order_type?

File: app/models/service/search.rb [hunk 3]
```diff
@@ -26,6 +26,7 @@ def search_data
       providers: providers.map(&:id),
       platforms: platforms.map(&:id),
       target_groups: target_groups.map(&:id),
+      service_type: [service_type] << offers.map(&:offer_type),
```
Comment (martaswiatkowska) [inline app/models/service/search.rb:29]: service doesn't have service_type in my scheme

**Pull request merged**